carp.studies.core/src/commonMain/kotlin/dk/cachet/carp/studies/infrastructure/StudyServiceRequest.kt

```diff
@@ -21,9 +22,19 @@ import kotlinx.serialization.Serializable
 sealed class StudyServiceRequest
 {
     @Serializable
-    data class CreateStudy( val owner: StudyOwner, val name: String, val invitation: StudyInvitation? = null ) :
+    data class CreateStudy( val owner: StudyOwner, val name: String, val description: String = "", val invitation: StudyInvitation? = null ) :
         StudyServiceRequest(),
-        ServiceInvoker<StudyService, StudyStatus> by createServiceInvoker( StudyService::createStudy, owner, name, invitation )
+        ServiceInvoker<StudyService, StudyStatus> by createServiceInvoker( StudyService::createStudy, owner, name, description, invitation )
+
+    @Serializable
+    data class SetInternalDescription( val studyId: UUID, val name: String, val description: String ) :
+        StudyServiceRequest(),
+        ServiceInvoker<StudyService, StudyStatus> by createServiceInvoker( StudyService::setInternalDescription, studyId, name, description )
+
+    @Serializable
+    data class GetStudyDetails( val studyId: UUID ) :
+        StudyServiceRequest(),
+        ServiceInvoker<StudyService, StudyDetails> by createServiceInvoker( StudyService::getStudyDetails, studyId )
```

Comment:
lot's of long lines :)

Reply:
These I won't break apart since it's boiler plate code either way.